### PR TITLE
Add audit doc

### PR DESCRIPTION
**Pull request by soltysh**
@janetkuo @sttts ptal

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/kubernetes.github.io/1168)

### CODE REVIEW

File: docs/admin/audit.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Configuration
+
+[Kube-apiserver](kube-apiserver) provides following options which are responsible
+for configuring where and how audit logs are handled:
+
+- `audit-log-path` - enables the audit log pointing to a file where the requests are being logged to.
+- `audit-log-maxage` - specifies maximum number of days to retain old audit log files based on the timestamp encoded in their filename.
+- `audit-log-maxbackup` - specifies maximum number of old audit log files to retain.
+- `audit-log-maxsize` - specifies maximum size in megabytes of the audit log file before it gets rotated. Defaults to 100MB
```
Comment (philips) [inline docs/admin/audit.md]: by "rotated" does it mean the old log gets deleted or is there some naming semantic used inside the path?

Comment (soltysh) [inline docs/admin/audit.md]: `audit-log-maxbackup` specifies how many old files are kept, old files have a timestamp appended at the end. By default there's not limit set on that amount.

File: docs/admin/audit.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - source ip of the request
+    - HTTP method being invoked
+    - original user invoking the operation
+    - impersonated user for the operation
+    - namespace of the request or <none>
+    - uri is the full URI as requested
+2. The response line containing:
+    - the unique id from 1
+    - response code
+
```
Comment (sttts) [inline docs/admin/audit.md]: How about an example?

File: docs/admin/audit.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+comming to the server. Each audit log contains two entries:
+
+1. The request line containing:
+    - unique id allowing to match the response line (see 2)
+    - source ip of the request
+    - HTTP method being invoked
+    - original user invoking the operation
+    - impersonated user for the operation
+    - namespace of the request or <none>
+    - uri is the full URI as requested
```
Comment (sttts) [inline docs/admin/audit.md]: sentence sounds strange, better: `s/uri is //`

File: docs/admin/audit.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+* TOC
+{:toc}
+
+## Audit in Kubernetes
+
+Kubernetes currently provides only basic audit capabilities, there is still a lot
+of work going on to provide fully featured auditing capabilities (see https://github.com/kubernetes/features/issues/22).
+
+Kubernetes audit is part of [kube-apiserver](kube-apiserver) logging all requests
```
Comment (sttts) [inline docs/admin/audit.md]: Are those links supposed to work?

Comment (soltysh) [inline docs/admin/audit.md]: Yes they, should but it should be `/docs/admin/kube-apiserver`.

**Comment by soltysh** [comment]
Comments addressed, I've additionally added a few words explaining log rotation. ptal

**Comment by sttts** [comment]
lgtm

**Comment by soltysh** [comment]
Rebased.

**Comment by devin-donnelly** [comment]
Is this change targeting the 1.4 release, or can it be published immediately?

**Comment by soltysh** [comment]
It's definitely part of 1.4 release, and since it's part of k8s for over a month already can be published immediately as well.

**Comment by devin-donnelly** [comment]
This looks like a duplicate of Pull Request #1193 . I'm going to close this one. We'll merge the other one.

**Pull request closed without merging**